ebuild-writing/common-mistakes/text.xml

```diff
@@ -107,18 +114,20 @@ preferred.</note>
 <p>
 "-Werror" is a flag which turns all warnings into errors and thus will abort
 compiling if any warning is encountered.
+See <uri link="https://bugs.gentoo.org/show_bug.cgi?id=werror">this tracker</uri>
```

Comment:
This should refer to "bug 260867" in the text and in the link, for consistency with the bug reference above.